.csscomb.json

```diff
@@ -1,24 +1,319 @@
 {
-    "remove-empty-rulesets": true,
     "always-semicolon": true,
-    "color-case": "lower",
     "block-indent": "    ",
+    "color-case": "lower",
     "color-shorthand": true,
     "element-case": "lower",
     "eof-newline": true,
-    "leading-zero": false,
+    "leading-zero": true,
     "quotes": "double",
+    "tab-size": true,
+    "remove-empty-rulesets": true,
+    "strip-spaces": true,
+    "unitless-zero": true,
+    "vendor-prefix-align": true,
+    "space-between-declarations": "\n",
     "space-before-colon": "",
     "space-after-colon": " ",
     "space-before-combinator": " ",
     "space-after-combinator": " ",
-    "space-between-declarations": " ",
     "space-before-opening-brace": " ",
     "space-after-opening-brace": "\n",
-    "space-after-selector-delimiter": " ",
     "space-before-selector-delimiter": "",
+    "space-after-selector-delimiter": "\n",
     "space-before-closing-brace": "\n",
-    "strip-spaces": true,
-    "tab-size": true,
-    "unitless-zero": true
+    "sort-order": [
+        "$variable",
```

Comment:
I'd keep them on top, but I think we need the third opinion here) So far it feels to me like trying to avoid local variables whenever possible and defining them all in `default.scss` is the way we are supposed to implement. So it won't have much effect anyway.